igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import libvirt, sys
+
+def main():
+    URI = "qemu:///system"
+    strpoolname = 'xen-data'
+    strvolname = 'test-vol'
```

Comment:
Why hardcode? It would be better to have it as parameter and get it from Puppet.